LibSWBF2/Chunks/GenericChunk.cpp

```diff
@@ -254,4 +262,20 @@ namespace LibSWBF2::Chunks
 	template struct LIBSWBF2_API GenericChunk<"RTYP"_m>;
 	template struct LIBSWBF2_API GenericChunk<"BNAM"_m>;
 	template struct LIBSWBF2_API GenericChunk<"DTLX"_m>;
-}
\ No newline at end of file
+
+	template struct LIBSWBF2_API GenericChunk<1600943724>;
+	template struct LIBSWBF2_API GenericChunk<1684828791>;
+	template struct LIBSWBF2_API GenericChunk<1701667150>;
+
+#ifndef _WIN32
+	//Undef ref linker errors fix
+	//template struct LIBSWBF2_API GenericChunk<"XFRM"_m>;
+	//template struct LIBSWBF2_API GenericChunk<"inst"_m>;
+	//template struct LIBSWBF2_API GenericChunk<"DTEX"_m>;
```

Comment:
Sloppy, might cause problems on your end w/VS.  Also, I've been wondering if we should move these instantiations to their respective source files.  If we did this, it would make it easier to compile individual modules of this library (e.g. libSWBF2 for MSH processing only, without LVL API/Chunks).  The finer points of linking are still hazy to me.  I'm not sure if executables that statically link the entire libSWBF2 library, but only use MSH related functions, still have LVL related code/symbols embedded in their binaries...

Reply:
Nope, if you link statically only used code get's copied

Reply:
If we just want to compile a subset of functionality at some point in the future, this will be worth looking into. For now, just leave it there. IIRC there also were some linkink issues with the template instantiations declared in their respective cpp's.

Reply:
Please use the string representation as template parameter instead of putting in the integer, it's more readable.
(e.g. `<"wrld"_m>` instead of `<1684828791>`)

Reply:
Yea I just copied those from the linker errors, fixed now.